app/components/syntax-highlighted-diff.ts

```diff
@@ -166,8 +168,12 @@ export default class SyntaxHighlightedDiffComponent extends Component<Signature>
     );
 
     highlighterPromise.then((highlighter) => {
-      highlighter.loadLanguage(this.args.language as shiki.Lang).then(() => {
-        this.asyncHighlightedHTML = highlighter.codeToHtml(this.codeWithoutDiffMarkers, { lang: this.args.language });
+      highlighter.loadLanguage(this.args.language as shiki.BundledLanguage | shiki.LanguageInput | shiki.SpecialLanguage).then(() => {
+        this.asyncHighlightedHTML = highlighter.codeToHtml(this.codeWithoutDiffMarkers, {
+          lang: this.args.language,
+          theme: this.args.theme,
```

Comment:
@libmartinito it looks like there's `isDarkMode` already being used here, couldn't we infer theme from that? We only support one theme for dark vs. light mode at the moment, seems unnecessary to have this passed in (it wouldn't even work if you passed in a separate theme since `getOrCreateCachedHighlighterPromise` would be called with a fixed theme anyway)

Reply:
it seems that there's no way to infer the theme. in the [docs](https://shiki.matsu.io/guide/install#highlighter-usage), it says that the theme should be defined. updated to use the theme defined in the same component

Reply:
Yep I meant that the component (`SyntaxHighlightedDiff`) "infers" based whether dark mode is set or not